Source/WTF/wtf/Compiler.h

```diff
@@ -447,25 +447,33 @@
     _Pragma(_COMPILER_STRINGIZE(compiler diagnostic push)) \
     _COMPILER_CONCAT(IGNORE_WARNINGS_BEGIN_IMPL_, cond)(compiler, warning)
 
+/* condition is either 1 (true) or 0 (false, do nothing) */
```

Comment:
WebKit comments are sentence style with a capital first letter and a period at the end.

Reply:
Oops, fixed.